### PR TITLE
Document UTF-8 encoding input plugin support

### PR DESCRIPTION
**Pull request by nigels-com**
In relation to https://github.com/fluent/fluent-bit/pull/1668

### CODE REVIEW

File: configuration/encoder.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Encoding input to UTF-8
+
+Some input plugins converting input to UTF-8 from a specified encoding.  The current set of supported encodings are:
```
Comment (nigels-com) [inline configuration/encoder.md]: converting -> convert

File: input/head.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ The plugin supports the following configuration parameters:
 | Key | Rename a key. Default: head. |
 | Lines | Line number to read. If the number N is set, in\_head reads first N lines like head\(1\) -n. |
 | Split\_line | If enabled, in\_head generates key-value pair per line. |
+| Encoder | Optionally encode input to UTF-8. E.g. `iso-8859-1` |
```
Comment (nigels-com) [inline input/head.md]: Link to configuration/encoder.md

**Comment by edsiper** [comment]
note: on hold until  https://github.com/fluent/fluent-bit/pull/1668 gets merged

**Comment by edsiper** [comment]
closing due to timeout, let's reopen once https://github.com/fluent/fluent-bit/pull/1668 is ready

**Pull request closed without merging**